jgrapht-core/src/main/java/org/jgrapht/generate/GridGraphGenerator.java

```diff
@@ -71,13 +71,13 @@ public GridGraphGenerator(int rows, int cols)
     @Override
     public void generateGraph(Graph<V, E> target, Map<String, V> resultMap)
     {
-        Map<Integer, V> map = new TreeMap<>();
+        List<V> list = new ArrayList<>();
```

Comment:
Preallocate memory in the array list by specifying its size. Also, consider renaming this variable to something link `vertices` or `vertexList` (the previous name `map` wasn't good either, so let's improve this).

Reply:
Agree.
I changed the name to vertices